pyrene/src/components/Table/TableHeader/TableHeaderCell.jsx

```diff
@@ -26,8 +26,7 @@ const getIconComponent = (className) => {
 const TableHeaderCell = (props) => (
   <div
     onClick={(event) => props.toggleSort(event)}
-    styleName={clsx({ multiSelect: props.multiSelect, tableHeaderCell: true })}
-    className={clsx(props.className, 'unSelectable')}
+    className={clsx({ [styles.multiSelect]: props.multiSelect, [styles.tableHeaderCell]: true }, props.className, 'unSelectable')}
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
className={clsx({ [styles.multiSelect]: props.multiSelect, styles.tableHeaderCell }, props.className, 'unSelectable')}
```